paper-input-error.html

```diff
@@ -65,20 +65,5 @@
     behaviors: [
       Polymer.PaperInputAddonBehavior
     ],
-
-    properties: {
-      /**
-       * True if the error is showing.
-       */
-      invalid: {
```

Comment:
Well, from this perspective yes. The question is, why did this property was introduced? I think the answer is to style itself only, which is IMHO unoptimized. As the owner of the component, you are the one to take responsibility for the optimization change I proposed. I mean, a lot of PRs won't never be accepted, I wouldn't mind don't worry! :)